word_search.py

```diff
@@ -0,0 +1,102 @@
+'''
+Created on 9 June 2017
+
+@author: Eva
+'''
+from queue import Queue
+
+class WordSearch:
+    
+    def __init__(self):
+        return
+        
+    def words_found_in_grid(self, rowNo, columnNo, grid, dictionary):
+        """Uses bfs to find valid words from a grid according to a given dictionary 
+        
+            Args:
+                rowNo: number of rows in the grid
+                columnNo: number of columns in the grid
+                grid: two-dimensional array of characters
```

Comment:
Yes, I think it is okay to make some (documented) assumption.
In this case it is totally reasonable to assume that all rows have the same number of columns.